README.md

```diff
@@ -84,6 +84,63 @@ python run.py
 deactivate
 ```
 
+### Run app in Linux
+
+The project runs on Python 3.
+
+1. Create a virtual enviorntment:
+`$ virtualenv`
```

Comment:
Change this line to `virtualenv venv`. So, that we don't deviate from the Windows setup instructions.